QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeFull.java

```diff
@@ -17,14 +17,21 @@ public AutonomousModeFull() {
 	@Override
 	protected void runAutonomous() throws InterruptedException,
 			AutonomousModeOverException {
+		setClampHeight(1.0f);
 		collectTote();
+		setClampHeight(0.0f);
+		QuasarHelios.clamp.openControl.set(true);
 		drive(nudge.get());
-		// TODO: Forklift container
+		QuasarHelios.clamp.openControl.set(false);
+		setClampHeight(1.0f);
 		// TODO: Hold container with top claw
 		drive(toteDistance.get());
 		collectTote();
+		setClampHeight(0.0f);
+		QuasarHelios.clamp.openControl.set(true);
 		drive(nudge.get());
-		// TODO: Forklift container
+		QuasarHelios.clamp.openControl.set(false);
```

Comment:
Maybe `QuasarHelios.clamp.openControl.set` could have a good helper addition to the base class?